spec/models/bet_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe Bet, type: :model do
+  let(:bob)   { User.create(username: 'Bob')   }
+  let(:peter) { User.create(username: 'Peter') }
+  let(:jack)  { User.create(username: 'Jack')  }
+
+  let(:bet1) do
+    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!')
+  end
+
+  let(:option1) { BetOption.create(option_text: 'Atlanta wins') }
+  let(:option2) { BetOption.create(option_text: 'New England wins') }
+
+  let(:ub1) do
+    UserBet.create(user:       bob,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 100)
+  end
+
+  let(:ub2) do
+    UserBet.create(user:       peter,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 50)
+  end
+
+  let(:ub3) do
+    UserBet.create(user:       jack,
+                   bet:        bet1,
+                   bet_option: option2,
+                   amount_bet: 300)
+  end
+
+  let(:ub) do
+    ub1
+    ub2
+    ub3
+  end
+
+  context_string = 'Bet: Superbowl - Atlanta vs New England;\n' \
+    'Betters on Atlanta: Bob ($100), Peter ($50);\n' \
+    'Betters on New England: Jack ($300).'
+
+  context context_string do
+    describe('#winner_total') do
+      subject { bet1.winner_total }
+      context 'Atlanta wins' do
+        it 'returns the total amount of money bet on Atlanta winning (150)' do
+          ub
+          option1.update(winner: true)
+          expect(subject).to eq 150
```

Comment:
This is what we call a "magic number" we try and avoid having numbers floating around in code without any context, because its very easy to become confused about what they represent.

This should be encapsulated in a `let` statement ..... something like `let(:expected_debt_amount) { ... }